managed/CounterStrikeSharp.API/Modules/Admin/RequiresPermissionsOr.cs

```diff
@@ -11,31 +11,28 @@ public class RequiresPermissionsOr : BaseRequiresPermissions
     {
         public RequiresPermissionsOr(params string[] permissions) : base(permissions) { }
 
-        public override bool CanExecuteCommand(CCSPlayerController? caller)
+        public override bool CanExecuteCommand(SteamID? steamID)
         {
-            if (caller == null) return true;
-            if (AdminManager.PlayerHasCommandOverride(caller, Command))
+            if (steamID == null) return true;
```

Comment:
is this reliable?

Reply:
It should be. SteamID null checks are peppered throughout core AdminManager functions, including PlayerHasCommandOverride itself, and I haven't seen any bug reports these functions failing.